beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -578,14 +695,24 @@ func (f *blocksFetcher) fetchBlobsFromPeer(ctx context.Context, bwb []blocks.Blo
 			continue
 		}
 		f.p2p.Peers().Scorers().BlockProviderScorer().Touch(p)
-		robs, err := verifyAndPopulateBlobs(bwb, blobs, req, f.bs)
-		if err != nil {
+		if err := verifyAndPopulateBlobs(bwb, blobs, req, f.bs); err != nil {
```

Comment:
why do we have f.p2p.Peers().Scorers().BlockProviderScorer().Touch(p) before we verify the blob?

Reply:
I prefer not to touch this thing in this PR, even if I agree with you.
Let's do this in a separated PR.